docs/source/workflows/design_workflows.rst

```diff
@@ -212,3 +212,33 @@ Thus, archiving all the design workflows contained within a branch will result i
 
     # Restore the branch to active status.
     my_branch = project.branches.restore(my_branch.uid)  # my_branch.archived == False
+
+You can also update the data on a branch in a similar way that one updates to later date via the web application.
```

Comment:
Kind of a confusingly worded sentence. Maybe "You can also update the data on a branch similarly to the web application."